### DISCUSSION TITLE
Report Sharing

### DISCUSSION THREAD

**Discussion by ewallz** [ludeknovy/jtl-reporter#253] [Q&A]
Hi, i've generated the Scenario share link. But when other user open the link, they can only see the overall charts (Scenario Trends). The bottom part, list of all test run are not visible. Then I need to generate links for each test run for other users to see.

Is it possible to include all the Test Runs for that particular Scenario under a single Scenario share link? So that I just generate the Scenario link and user will have access to that Scenario report + All its Test Run results/graphs, etc...like I see on Admin side.

**Comment by ludeknovy** [comment]
Hi @ewallz. Why dont you create a new readonly user for this case?

**Reply by ewallz** [reply]
I did...but..
1. readonly users cannot access the 'scenario' page that has the summary graph (ontop) + run lists (below), when accessing the project's scenario, this notice shown; "Forbidden, you dont have permission to access this page."
2. readonly users can only access the Dashboard, Test Run lists, single test results page.
3. If possible i don't want user to login...just access using link+token for the Scenario & All test runs under it.

**Reply by ludeknovy** [reply]
1. That's most likely a bug, because I think that this should be allowed.
2. No, it should have access to the projects you allow it to.
3. Not sure if this is going to happen in the foreseeable future, as this would require a significant number of changes. But PR is welcomed.
